examples/src/main/scala/org/apache/spark/examples/ml/DeveloperApiExample.scala

```diff
@@ -169,10 +169,10 @@ private class MyLogisticRegressionModel(
     Vectors.dense(-margin, margin)
   }
 
-  /** Number of classes the label can take. 2 indicates binary classification. */
+  // Number of classes the label can take. 2 indicates binary classification.
```

Comment:
Yeah, good to make this a standard comment, not scaladoc style

Reply:
+1